src/server.ts

```diff
@@ -1,7 +1,9 @@
 import Pummiralli from './domain/pummiralli'
+import { HistoryEvent } from './domain/messages'
 import { createServer, Socket, AddressInfo } from 'net'
 import Log from './util/log'
 import { tryParseObject } from './util/parser'
+import fs from 'fs'
 
 // import Ajv from 'ajv'
 // import schema from './messages/schema.json'
```

Comment:
Should we remove all these schema thingies for now?

Reply:
👍